src/controllers/teamController.spec.js

```diff
@@ -0,0 +1,68 @@
+const Team = require('../models/team');
+const teamController = require('./teamController');
+const { mockReq, mockRes, assertResMock } = require('../test');
+const helper = require('../utilities/permissions');
+
+const makeSut = () => {
+    const { postTeam } = teamController(Team);
+    return {
+        postTeam,
+    };
+};
+
+jest.mock('../utilities/permissions', () => ({
```

Comment:
It this line of code necessary? In the function below (mockHasPermission) we are already creating a mock version of the hasPermission function.